shared/tinyusb/tusb_config.h

```diff
@@ -59,8 +61,9 @@
 
 // CDC Configuration
 #if CFG_TUD_CDC
-#define CFG_TUD_CDC_RX_BUFSIZE  (256)
-#define CFG_TUD_CDC_TX_BUFSIZE  (256)
+#define CFG_TUD_CDC_EP_BUFSIZE  ((CFG_TUD_MAX_SPEED == OPT_MODE_HIGH_SPEED) ? 512 : 256)
```

Comment:
This is incorrect and will reintroduce a bug.  See ec1eeccab45af6893f19cd07292808bbe7c62cf5

I think it's better to just leave this variable unset, and let TinyUSB set it.

Reply:
In the commit message you referenced:

> The value of CFG_TUD_CDC_EP_BUFSIZE should match the endpoint size,

If I remove the above, it will Not match the EP size (512), sure ?

Reply:
Removed.